docs/getting-started/linux-installer.md

```diff
@@ -109,6 +109,17 @@ modified as needed. Possible configuration options can be found in the
 - [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector)
 - [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
 
+To use an existing collector configuration file instead of the default, run
```

Comment:
I'd write: 

the installer script with the `--collector-config PATH_TO_CONFIG` option, replacing `PATH_TO_CONFIG` with the absolute path to the desired configuration file on the host. For example: